src/plugins/raven/services/Raven.js

```diff
@@ -41,6 +43,8 @@ export default class Raven {
     this.mqtt = mqtt;
     this.store = store;
     this.apiEndpoint = apiEndpoint;
+    this.logStreamRequestCount = 0;
+    window.onerror = this.onError.bind(this);
```

Comment:
@cohencodes nit: i would put this at the end of the constructor instead of in the middle of all the `this` assignments

Reply:
fixed